bin/znapzend

```diff
@@ -71,6 +72,39 @@ sub main {
         }
     }
 
+    if (defined($opts->{forbidDestRollback})) {
+        $opts->{forbidDestRollback} = 1;
+    } else {
+        $opts->{forbidDestRollback} = 0;
+    }
+
+    if (defined($opts->{sinceForced}) && ($opts->{sinceForced} eq '')) {
+        delete $opts->{sinceForced};
+    }
+
+    if (defined($opts->{since}) && ($opts->{since} eq '')) {
+        delete $opts->{since};
+    }
```

Comment:
is this actually a problem ?

Reply:
Did not check really, just for clarity in other code - if we have an input and go this way, the input makes sense at least for basics. Especially as we set this value from two options, so someone might (contrived) write `--since=X --sinceForced` thinking it is an option like that (probably getopt would protect from this one but still).

Reply:
if it comes from getopt, then you can only have an empty string it it is something with which accepts a value ... so for 'since' that makes sense, but then you should also complain I think ... for since forced it does not as it does not accept a value in the first place ... 

the reason I am after these kinds of issues is becuase the more of them are in the code the harder it becomes to see the logic of the code and these unnecessary safeguards obscure the view ... 

this is why I like openapi for REST interfaces ... then I can be sure the openapi implementation sanitizes and validates the input before I go at it and I do not have to write extra safeguard and cleaning code locally ... 

getopt works in a similar way

Reply:
Actually `--sinceForced=X` should work (and was tested to) as an explicitly dangerous replacement to `--since`. Maybe more conflict-checks and complaints make sense here. Technically still one can pass `--arg=''` to get an empty value in, and we don't want to bother about these.

Reply:
ok